src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.controller;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dto.LimitedPartnershipSubmissionCreatedResponseDto;
+import uk.gov.companieshouse.limitedpartnershipsapi.utils.ApiLogger;
+
+import java.net.URI;
+
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.URL_PARAM_TRANSACTION_ID;
+
+@RestController
+@RequestMapping("/transactions/{" + URL_PARAM_TRANSACTION_ID + "}/limited-partnership/partnership")
+public class PartnershipController {
+
+    @PostMapping
+    public ResponseEntity<Object> createPartnership(
+            HttpServletRequest request) {
+
+        ApiLogger.debug("createPartnership");
+
+        URI location = URI.create("/transactions/12321123/limited-partnership/partnership/3235233232");
+        LimitedPartnershipSubmissionCreatedResponseDto response = new LimitedPartnershipSubmissionCreatedResponseDto();
+        response.setId("3235233232");
```

Comment:
Could we maybe change this to a value like "NOT-CREATED"?

Reply:
I could but it won't really change anything as once the story is complete it should return the real value created by the service. Are you currently depending on the the id being NOT-CREATED? I can change it if you are.

Reply:
No I'm not. Can live with this.